### ISSUE TITLE
Calendar | Today doesn't receive focus

### ISSUE THREAD

**Issue by cetincakiroglu** [primefaces/primeng#14256]
Steps to reproduce:

- Enable inline mode, focus on next button.
- Press tab, next focusable element in the page receives focus instead of today.
